Assets/MixedRealityToolkit.Tools/ReserializeAssetsUtility/ReserializeAssetsUtility.cs

```diff
@@ -30,6 +30,7 @@ private static void ReserializeMaterials()
             AssetDatabase.ForceReserializeAssets(array);
         }
 
+        [MenuItem("Mixed Reality Toolkit/Utilities/Reserialize/Reserialize Selection")]
```

Comment:
Maybe put this in separate PR? Seems unrelated to this change

Reply:
I guess it's not necessary but seems severe to remove from this PR and create a whole separate one for it and issue item